### DISCUSSION TITLE
Daily Notes Opener

### DISCUSSION THREAD

**Discussion by NikosDevPhp** [TfTHacker/DashboardPlusPlus#11] [Ideas]
Can I somehow have a link in the home dashboard to open daily notes?

Thank you so much for this repo

**Comment by NikosDevPhp** [comment]
`="[[" + dateformat(date(today), "yyyy-MM-dd") + "|Daily Note ]]"`

**Comment by who-took-hu4d** [comment]
Hey @NikosDevPhp !

Sorry to necro this super old thread, but I'm just seeing it now.
I use the Obsidian plugin [Buttons](https://github.com/shabegom/buttons) to achieve this.

You've gotta be a little bit funky with the markdown since neither Dashboard++ nor Buttons really wants you to do this though!
The following markdown code
```markdown
---
cssclass: dashboard
---
# Personal
- &nbsp;
	```button
	name Open Daily Note
	type command
	action Daily notes: Open today's daily note
	color blue
	```
	^button-dailyNote

	```button
	name Search
	type command
	action Omnisearch: Vault search
	color blue
	```
	^button-Search
 
	```button
	name Start Meeting
	type command
	action Create Note in Folder: newMeetingNote
	color blue
	```
	^button-newMeetingNote
- Sites
	- ...
	- ...
	- ...
	- ...
	- ...
	- ...

- Potential Purchases
	- ...
	- ...
	- ...
...
```

is able to put the Buttons into the columns as seen below:

![image](https://github.com/TfTHacker/DashboardPlusPlus/assets/128825691/5f8eaa1c-1bfc-4ec3-8d15-b6d7e0438b7e)

Hope this helps and that the email you get from this isn't too annoying lol

(Also, the Obsidian theme is [Border](https://github.com/Akifyss/obsidian-border) if you're interested)

**Reply by NikosDevPhp** [reply]
Hey man, thank you very much. I will try to do sth similar. Thanks
